### PR TITLE
[full-ci] [tests-only] Remove unimplemented acceptance tests

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Removal of unimplemented WebUI acceptance tests.
Removed tests scenarios have been listed in the issue https://github.com/owncloud/web/issues/7264
Also, pipelines are re-structured by changing the number of suites in each pipeline and some new pipelines are added, to balance out the time through all pipelines.

## Related Issue

- Part of https://github.com/owncloud/QA/issues/765#issuecomment-1247646666
- Fixes https://github.com/owncloud/web/issues/7215

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic2</strong> https://drone.owncloud.com/owncloud/web/28353/14/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIWebdavLockProtection-upload_feature-L32.png
  

### webUIWebdavLockProtection-upload_feature-L33.png
  


</p></details>

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature [hunk 1]
```diff
@@ -25,17 +25,6 @@ Feature: files and folders can be deleted from the trashbin
...
-  @issue-1725 @smokeTest @ocisSmokeTest
-  Scenario: Delete files and check that they are gone
-    When the user deletes file "lorem.txt" using the webUI
-    And the user deletes file "sample,1.txt" using the webUI
-    And the user opens folder "simple-folder" using the webUI
-    Then file "lorem.txt" should not be listed on the webUI
-    And file "sample,1.txt" should not be listed on the webUI
-    But file "lorem.txt" should be listed in the trashbin folder "simple-folder" on the webUI
-    And file "lorem-big.txt" should not be listed in the trashbin folder "simple-folder" on the webUI
-    But file "lorem-big.txt" should be listed on the webUI
```
Comment (saw-jan) [inline tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:37]: This scenario should be just like the one below it and should work

File: tests/acceptance/features/webUIFilesSearch/search.feature [hunk 2]
```diff
@@ -59,38 +59,6 @@ Feature: Search
     #And file "lorem-big.txt" with path "/strängé नेपाली folder" should be listed in the search results in the other folders section on the webUI
     #But file "lorem.txt" with path "/simple-folder" should not be listed in the search results in the other folders section on the webUI
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesSearch/search.feature:60]: Most of the other scenarios are also commented. Maybe they can be removed too? Do they even work as expected? We also have some search-related tests for e2e as well

Comment (grgprarup) [reply tests/acceptance/features/webUIFilesSearch/search.feature:60]: Those commented steps are not implemented and we are adding related tests in e2e so these steps can be removed.

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:37]: IMO, better remove this scenario.
Also the scenario below doesn't make sense in this feature file.
There might be similar features in `deleteFilesFolders.feature` file

Comment (grgprarup) [reply tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:37]: Thanx. I didn't notice that. They are in `deleteFilesFolders.feature`.

Comment (grgprarup) [reply tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:37]: The scenarios below are for the deletion of folders from the trash bin. I will change the scenario description.

Comment (saw-jan) [reply tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:37]: oh yeah, I didn't look in the background.

File: .drone.star [hunk 3]
```diff
@@ -71,7 +71,6 @@ config = {
             "suites": {
                 "oC10Basic1": [
                     "webUIAccount",
-                    "webUIWebdavLocks",
```
Comment (saw-jan) [inline .drone.star:74]: This pipeline has too few tests. Please add other suite to balance out the time.

Long pipelines:
```
SharingAccept
Files5
SharingPublic2
```

**Comment by SagarGi** [comment]
@grgprarup  does removing showing promising result to balance out the time in drone? if you could provide before and after time after the adjustment.

**Comment by grgprarup** [comment]
> @grgprarup does removing showing promising result to balance out the time in drone? if you could provide before and after time after the adjustment.

Yeah, after removal of undefined step has decreased lots of time,

File: .drone.star [hunk 4]
```diff
@@ -463,48 +461,50 @@ config = {
...
+                    "webUISharingFilePermissionsGroups",
                     "webUISharingFilePermissionMultipleUsers",
                     "webUISharingFolderPermissionMultipleUsers",
                     "webUISharingFolderAdvancedPermissionMultipleUsers",
+                    "webUISharingFolderAdvancedPermissionsGroups",
                 ],
                 "oCISSharingAndUpload": [
+                    "webUIResharing1",
                     "webUIResharing2",
-                    "webUIUpload",
```
Comment (saw-jan) [inline .drone.star:493]: These suites have one 4 scenarios  in oCIS pipeline

File: .drone.star [hunk 5]
```diff
@@ -69,52 +69,46 @@ config = {
...
                     "webUIAccount",
-                    "webUIWebdavLocks",
-                ],
-                "oC10Basic2": [
+                    "webUILogin",
                     "webUIPrivateLinks",
                     "webUIWebdavLockProtection",
                 ],
-                "oC10FileOperations": [
+                "oC10FileFolderOperations1": [
```
Comment (kiranparajuli589) [inline .drone.star:78]:
```suggestion
                "oC10FileFolderOperations": [
```

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by SagarGi — COMMENTED** [review]
Promising result after removing in drone build time. LGTM :+1:

**Comment by SwikritiT** [comment]
LGTM

**Comment by phil-davis** [comment]
I rebased to resolve a conflict. Let's see the CI result (I think that I resolved the conflict easily), and then I will approve and merge.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**